lib/matplotlib/pyplot.py

```diff
@@ -2027,18 +2027,14 @@ def clim(vmin=None, vmax=None):
     """
     Set the color limits of the current image.
 
-    To apply clim to all axes images do::
```

Comment:
I don't think this is true. That call would only set the limit of the current image. Scaling multiple images is again described further below.